labs/lab3/lab3.adoc

```diff
@@ -63,23 +55,23 @@ Cluster administrators can view all of the nodes that have constitute the cluste
 master$ oc get nodes
 ----
 
-View all of the Projects that have been created by users or to support the platform:
+View all of the link:https://docs.openshift.com/container-platform/lastest/dev_guide/projects.html[Projects] that have been created by users or as part of the base infrastructure to support the platform:
 
 .master$
 [source, bash]
 ----
 master$ oc get projects
```

Comment:
The inclusion of "master$" here seems redundant. I thought the purpose of line 60 ".master$" was to set context, and then all the commands themselves would not be preceded by "master$", so that the student could cut and paste the commands easier.

Reply:
Removed all occurrences as recommended